src/core/plugins/json-schema-2020-12-samples/fn/class/OptionRegistry.js

```diff
@@ -4,7 +4,17 @@
 import Registry from "./Registry"
 
 class OptionRegistry extends Registry {
-  #defaults = {}
+  #defaults = {
+    randomEnabled: false,
+    random: Math.random,
+    minInt: 0,
```

Comment:
Let's align with json-schema-faker here:

```suggestion
    minInteger: -100000000,
```

Reply:
Ok